app/login.go

```diff
@@ -68,8 +68,8 @@ func (a *App) AuthenticateUserForLogin(id, loginId, password, mfaToken string, l
 
 	if pluginsEnvironment := a.GetPluginsEnvironment(); pluginsEnvironment != nil {
 		var rejectionReason string
-		pluginContext := &plugin.Context{}
 		pluginsEnvironment.RunMultiPluginHook(func(hooks plugin.Hooks) bool {
+			pluginContext := a.PluginContext()
```

Comment:
Why is this plugin context instantiated for each plugin, vs. the others which get instantiated once for all?

Reply:
Because I wasn't paying attention and just trusted where it was being instantiated before. Will fix.

Reply:
Good catch. Fixed.